src/chrome/content/rules/Steam.xml

```diff
@@ -14,40 +14,63 @@
 		<test url="http://api.steamcommunity.com/" />
 		<test url="http://cdn.steamcommunity.com/public/css/skin_1/header.css" />
 
+	<target host="steamgames.com" />
+		<test url="http://steamgames.com/" />
+	<target host="www.steamgames.com" />
+		<test url="http://www.steamgames.com/" />
 	<target host="partner.steamgames.com" />
+		<test url="http://partner.steamgames.com/" />
 
 	<target host="api.steampowered.com" />
 		<test url="http://api.steampowered.com/ISteamWebAPIUtil/GetSupportedAPIList/v1/" />
-	<target host="kgs.steampowered.com" />
 	<target host="partner.steampowered.com" />
+		<test url="http://partner.steampowered.com/" />
 	<target host="storefront.steampowered.com" />
 		<test url="http://storefront.steampowered.com/app/440/" />
 	<target host="store.steampowered.com" />
 		<test url="http://store.steampowered.com/app/440/" />
 	<target host="support.steampowered.com" />
+		<test url="http://support.steampowered.com/" />
 	<target host="help.steampowered.com" />
+		<test url="http://help.steampowered.com/" />
 	<target host="cdn.steampowered.com" />
 		<test url="http://cdn.steampowered.com/public/images/gift/steamcards_promo_02.png" />
 	<target host="cdn.store.steampowered.com" />
 		<test url="http://cdn.store.steampowered.com/public/images/gift/steamcards_promo_02.png" />
 	<target host="www.steampowered.com" />
+		<test url="http://www.steampowered.com/" />
+	<target host="media.steampowered.com" />
+		<test url="http://media.steampowered.com/" />
 
-	<securecookie host="^steamcommunity\.com$" name=".+" />
-	<securecookie host="^partner\.steamgames\.com$" name=".+" />
-	<securecookie host="^partner\.steampowered\.com$" name=".+" />
+	<target host="*.akamai.steamstatic.com" />
+		<test url="http://community.akamai.steamstatic.com/" />
+		<test url="http://store.akamai.steamstatic.com/" />
+		<test url="http://cdn.akamai.steamstatic.com/" />
+	<target host="*.edgecast.steamstatic.com" />
+		<test url="http://community.edgecast.steamstatic.com/" />
+		<test url="http://store.edgecast.steamstatic.com/" />
+		<test url="http://cdn.edgecast.steamstatic.com/" />
 
-	<rule from="^https?://cdn\.steamcommunity\.com/"
-			to="https://steamcommunity-a.akamaihd.net/" />
+	<securecookie host=".+" name=".+" />
 
-	<rule from="^https?://storefront\.steampowered\.com/"
+	<rule from="^http://(www\.)?steamgames\.com/"
 			to="https://store.steampowered.com/" />
 
-	<rule from="^https?://((cdn|cdn\.store|www)\.)?steampowered\.com/"
+	<rule from="^http://((cdn|storefront|www)\.)?steampowered\.com/"
```

Comment:
`steampowered.com` is not listed in your targets

Reply:
Good point. Tests didn't catch that.

Reply:
This rule does not work for deep links.

See http://storefront.steampowered.com/v/gfx/apps/210950/capsule_467x181.jpg

Reply:
I had a rule to cover `/v/gfx` before, but CI complained there was no test for it (even though there was). Do tests/rules work for covering paths and not just domains?

Also `/v/gfx` isn't directly used anywhere.

Reply:
> Do tests/rules work for covering paths and not just domains?

Yes, they do. Since you are force pushing, I can't really tell what was wrong.

Reply:
I've pushed it again so you can take a look.

Reply:
It's because it's already rewritten by the previous rule. You should try switching L40 and L43. That should help.

If that does not work, you will need to use a stricter rule but let's try this first.